salt/modules/ps.py

```diff
@@ -147,26 +147,22 @@ def top(num_processes=5, interval=3):
     for pid in psutil.pids():
         try:
             process = psutil.Process(pid)
-            user, system = process.cpu_times()
-        except ValueError:
-            user, system, _, _ = process.cpu_times()
         except psutil.NoSuchProcess:
             continue
+        else:
+            user, system = process.cpu_times()[:2]
         start_usage[process] = user + system
     time.sleep(interval)
     usage = set()
     for process, start in six.iteritems(start_usage):
-        try:
-            user, system = process.cpu_times()
-        except ValueError:
-            user, system, _, _ = process.cpu_times()
-        except psutil.NoSuchProcess:
-            continue
+        user, system = process.cpu_times()[:2]
```

Comment:
Yeah, I didn't think this would raise an exception until I later found that a function call below was doing the same thing and added a commit. Should have come back here and did the same, good catch. I just pushed a commit.